doc/cephadm/monitoring.rst

```diff
@@ -128,6 +128,24 @@ update its configuration:
 
 The ``reconfig`` command also sets the proper URL for Ceph Dashboard.
 
+Networks and Ports
+~~~~~~~~~~~~~~~~~~
+
+All monitoring services can have the network and port they bind to configured with a yaml service specification
+
+example spec file:
+
+.. code-block:: yaml
+
+    service_type: grafana
+    service_name: grafana
+    placement:
+      count: 1
+    networks:
+    - 192.169.142.0/24
+    spec:
+      port: 4200
```

Comment:
Is it also possible to specify the port in the CLI command `ceph orch apply grafana ...`?

Reply:
not at the moment. that would require adding a '--port' option to the generic 'ceph orch apply' command but that option would not be useful for all the services that can be applied with 'ceph orch apply'